pallets/prices/rpc/src/lib.rs

```diff
@@ -0,0 +1,106 @@
+//! RPC interface for prices pallet
+//!
+//! RPC installation: `rpc/src/lib.rc`
+//!
+//! Corresponding runtime API declaration: `pallets/prices/rpc/run-time/src/lib.rs`
+//! Corresponding runtime API implementation: `runtime/src/lib.rs`
+
+use jsonrpc_core::{Error as RpcError, ErrorCode, Result};
+use jsonrpc_derive::rpc;
+use minterest_primitives::{CurrencyId, Price};
+pub use prices_rpc_runtime_api::PricesApi as PricesRuntimeApi;
+use sp_api::ProvideRuntimeApi;
+use sp_blockchain::HeaderBackend;
+use sp_runtime::{generic::BlockId, traits::Block as BlockT};
+use std::sync::Arc;
+
+#[rpc]
+/// Base trait for RPC interface of prices
+pub trait PricesApi<BlockHash> {
+	/// This function returns a price for a currency.
+	/// If currency price has been locked, locked value will be returned.
+	/// Otherwise the value from Oracle RMS will be returned
+	///
+	///  - `&self` :  Self reference
+	///  - `at` : Needed for runtime API use. Runtime API must always be called at a specific block.
+	#[rpc(name = "prices_getCurrentPrice")]
+	fn get_current_price(&self, at: Option<BlockHash>, currency_id: CurrencyId) -> Result<Option<Price>>;
```

Comment:
Function signature has to be:
```suggestion
	fn get_current_price(&self, currency_id: CurrencyId, at: Option<BlockHash>) -> Result<Option<Price>>;
```